bin/wsim

```diff
@@ -91,6 +91,8 @@ prefix = ""
 if (args.lockstep or args.lockstepverbose or args.fcov or args.fcovimp):
     if (args.sim == "questa" or args.sim == "vcs"):
         prefix = "IMPERAS_TOOLS=" + WALLY + "/config/"+args.config+"/imperas.ic"
+        if not os.path.isfile(prefix): # If config is a derivative, look for imperas.ic in derivative configs
+            prefix = "IMPERAS_TOOLS=" + os.path.join(WALLY, "config", "deriv", args.config, "imperas.ic")
```

Comment:
Should probably use the same format for these paths for both the deriv and not deriv versions. I don't think it matters which, but probably better not two have 2 different versions of the same thing back to back.